contents/docs/new-to-posthog/getting-hogpilled.mdx

```diff
@@ -0,0 +1,134 @@
+
+<img height="700" alt="abstract-metrics-tree" src="https://res.cloudinary.com/dmukukwp6/image/upload/abstract_metrics_tree_c737164554.png" />
+
+**Events** are things that users do that we can track in code, and then measure across all usage of the product.
+
+## Example: ride sharing
+
+Uber can't make money if no one launches the app. `app_launched`, then, becomes, an essential event for them keep track of.
+
+But there's still no money until `ride_requested` happens.
+
+Even that doesn't quite get us to a ride. We need a couple more: `ride_accepted`  and `ride_began`
+
+<img width="1688" alt="ride-sharing-metrics-tree" src="https://res.cloudinary.com/dmukukwp6/image/upload/ride_sharing_metrics_tree_a0a1560337.png" />
+
+It's these events *together* that give us a picture of how people use the product to get a ride. While counting only `ride_began` would give us enough to measure our North Star, it would *not* be enough to learn how to influence it.
+
+## Funnels
+
+<img height="500" alt="ride-sharing-funnel" src="https://res.cloudinary.com/dmukukwp6/image/upload/ride_sharing_funnel_7e4f6b19eb.png" />
+
+Because events often happen in a predictable sequence, we can use that sequence to learn if users are getting stuck or bailing out.
+
+This is called a **funnel**.
+
+Funnels are simple: they measure successful progress between events. More people will launch the app than start a ride, but you still want to maximize the progress from one to the other. A funnel lets you understand how well that's going.
+
+If there's a sharp drop between, say, accepting a ride and starting one, that could point to an issue that needs to be fixed in the product.
+
+Funnels give us the information we need to diagnose problems and measure the impact of new solutions. We'll want to make sure we capture just enough event data to measure the critical path between starting in our product and succeeding with it.
+
+<Admonition icon="IconPencil" title="Planning a metrics tree">
+
+Recap your traction goal and North Star metric, then think about the events that drive that metric.
+
+These are the events that must be captured as part of integrating PostHog.
+
+</Admonition>
+
+# Integration planning
+
+Now:
+
+- We know what we're measuring  
+- We know why we're measuring it  
+- We know what events lead to the outcome we want
+
+With these details in-hand, we can make a plan to capture the data we need. This data collection will help us understand how users interact with our product, and how well our product meets our goals.
+
+<Admonition icon="IconWarning" title="Avoid event pitfalls" type="caution">
+
+Ideal event names are self-explanatory, avoiding strange abbreviations or codes. Don't stuff data into event names – they should be **comparable across all users** of your product. Instead, we want to tuck additional data into [properties](/docs/data/events#event-properties) *on* events.
```

Comment:
You sort of repeat yourself by talking about pitfalls and naming here, maybe just save for the gotchas/things to think about section